### PR TITLE
New draft to support more DC Nodes AND to support structured metadata

### PR DESCRIPTION
**Pull request by PonteIneptique**

### CODE REVIEW

File: capitains.rng [hunk 1]
```diff
@@ -139,25 +186,144 @@
       <text/>
     </element>
   </define>
-  
+  <define name="dc.contributor">
+    <element name="dc:contributor">
+      <a:documentation xml:lang="eng"> This documents hold contributor informations according to the
```
Comment (sonofmun) [inline capitains.rng]: "documents hold" doesn't make sense to me here. I would think it should be either "This documents" or "This holds", but not both.
This goes for the other new documentation strings as well.

Comment (sonofmun) [reply capitains.rng]: Also, "informations" should be "information". For whatever reason this is always a singular noun in English.

**Review by sonofmun — COMMENTED** [review]
Just a couple of irregularities with the English of the new documentation strings.
Otherwise I think it is good to go.

**Review by balmas — APPROVED** [review]
Nothing to add to @sonofmun 's comments.

File: capitains.rng (hunk 1 above)
Comment (PonteIneptique) [reply capitains.rng]: @sonofmun could you actually correct them ? I am fine with you appearing in the git log ;)

**Pull request merged**